packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Outbound/XCA ITI-39 Processor/destinationConnector-XCA Endpoint-responseTransformer-step-1-Case 1 - Success or PartialSuccess.js

```diff
@@ -63,19 +63,25 @@ if ('Success' == queryResponseCode.toString() || 'PartialSuccess' == queryRespon
 				var documentEncodedString = entry.*::Document.toString();
 				var decoded = FileUtil.decode(documentEncodedString);
 				var decodedAsString = Packages.java.lang.String(decoded);
-				
+
+        if (!decodedAsString) {
+          var errorMessage = 'Error with decoded document';
+          logger.error(errorMessage + fileName);
+          throw new Error(errorMessage);
+        }
+
 				// Parse the document header for some metadata
 				try {
 					var title = decodedAsString.split("<title>")[1].split("</title>")[0];
```

Comment:
I thought we were going to also fix this? Like, if we can't get `<title>` then keep processing without it? Currently it will throw to call `.split` if `<title>` is not on the string.